dll/ntdll/rtl/hashtable.c

```diff
@@ -0,0 +1,1264 @@
+// Placeholder for RTL_DYNAMIC_HASH_TABLE structure and related functions
+// For tests, create a separate file per each function under
+// .\reactos\modules\rostests\apitests\ntdll
+// TODO: Write tests per public function
+
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS NT User-Mode DLL
+ * FILE:            lib/ntdll/rtl/hashtable.c
```

Comment:
Please rename your file from
hashtable.c
to
hashtbl.c
so that it will be 8.3.
**Please wait with that step until all other review comments from other people have been addressed!**